src/db/models/attachmentPrototype.ts

```diff
@@ -37,15 +37,35 @@ const FIELDS = t.array(
 export const ATTACHMENT_PROTOTYPE_VALUE = t.intersection([
   // Required Fields
   t.type({
-    hasMeasures: t.number,
+    /**
+     * Some prototypes incorrectly quote numbers, which we need to account for
+     *
+     * TODO: clean prototype data and make type stricter
+     */
+    hasMeasures: t.union([
+      t.literal(0),
+      t.literal(1),
+      t.literal('0'),
+      t.literal('1'),
+    ]),
     name: LOCALIZED_STRING,
     entities: t.array(t.string),
   }),
   // Optional Fields
   t.partial({
     measureFields: FIELDS,
-    min: t.number,
-    max: t.number,
+    /**
+     * Some prototypes incorrectly quote numbers, which we need to account for
+     *
+     * TODO: clean prototype data and make type stricter
```

Comment:
Also, if possible, make sure it no longer happens. It will be validated by v4 models, but old models will still allow for this mixture (or is this a relict of a past?). Once admin command is in there, it could be ran periodically maybe, but problem should be truly solved.

It could also be achieved if old model that works with this data is replaced by v4 model.